README.md

```diff
@@ -172,7 +177,7 @@ You can [pass props to customize the API reference](https://github.com/scalar/sc
 
 The API Reference package is written in Vue, that shouldn’t stop you from using
 it in React though! We have created a client side (untested on SSR/SSG) wrapper
```

Comment:
consider moving this out of and into some kind of warning admonition:

<details>
<summary>Details</summary>

<img width="515" alt="image" src="https://github.com/scalar/scalar/assets/45884264/b19f39dd-fcfc-4233-992b-70498595b9cf">

</details>